src/odemis/acq/stream/_static.py

```diff
@@ -417,148 +433,43 @@ def dis_bbtl(v):
                     return float("inf")  # for None, None
             self.point.value = min(sempositions, key=dis_bbtl)
 
-        # no need for init=True, as Stream.__init__ will update the image
-        self.point.subscribe(self._onPoint)
-
-        # polarization VA
         # check if any polarization analyzer data, (None) == no analyzer data (pol)
         if self._pos.keys()[0][-1]:
             # use first entry in acquisition to populate VA (acq could have 1 or 6 pol pos)
-            self.polarization = model.VAEnumerated(self._pos.keys()[0][-1],
-                                choices=polpositions)
-
-        if self._pos.keys()[0][-1]:
-            self.polarization.subscribe(self._onPolarization)
+            self.polarization = model.VAEnumerated(self._pos.keys()[0][-1], choices=polpositions)
+
+            # Add a polarimetry VA containing the polarimetry image results.
+            # Note: Polarimetry analysis are only possible if all 6 images per ebeam pos exist.
+            # Also check if arpolarimetry package can be imported as will be private.
+            if len(polpositions) > 1 and arpolarimetry:
+                polarimetrypositions = {"DS0", "DS1", "DS2", "DS3",
```

Comment:
I think it'd probably make sense to have this list hardcoded somewhere the same way as POL_POSITIONS.